spec.yaml

```diff
@@ -54,6 +56,80 @@ securityDefinitions:
 
 
 paths:
+  /v4/info/:
+    get:
+      operationId: getInfo
+      tags:
+        - info
+      summary: Get information on the installation
+      description: |
+        Returns a set of details on the installation. The output varies based
+        on the provider used in the installation.
+
+        This information is useful for example when creating new cluster, to
+        prevent creating clusters with more worker nodes than possible.
+
+        ### Example for an AWS-based installation
+
+        ```json
+        {
+          "installation_name": "shire",
+          "provider": "aws",
+          "datacenter": "eu-central-1",
+          "workers_per_cluster": {
+            "max": 20,
+            "default": 3
+          },
+          "worker_instance_type": {
+            "options": [
+              "m3.medium", "m3.large", "m3.xlarge"
+            ],
+            "default": "m3.large"
+          },
+          "cpu_cores_per_worker": null,
+          "ram_size_gb_per_worker": null,
+          "storage_size_gb_per_worker": null
```

Comment:
I personally do not like this. It is nice to have a flat object but the naming of the object's properties is really not beneficial. I personally prefer having structured objects where properties are grouped based on their relationships or scopes. For instance it would make a lot of sense to me to have a separate block for all worker information.